common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -755,7 +754,11 @@ public void run() {
                             final LocalBroadcaster.IReceiverCallback resultReceiver = new LocalBroadcaster.IReceiverCallback() {
                                 @Override
                                 public void onReceive(@NonNull PropertyBag dataBag) {
-                                    completeInteractive(dataBag);
+                                    try {
+                                        completeInteractive(dataBag);
+                                    } catch (final Exception e) {
+                                        LocalBroadcaster.INSTANCE.registerExceptionDuringCallback(RETURN_AUTHORIZATION_REQUEST_RESULT, e);
```

Comment:
I'm actually not sure if we really need LocalBroadcaster to hold this exception for us. Can we just not hold it locally here in CommandDispatcher beginInteractive method and then throw it later?

Reply:
Removed, went with a final array locally in CommandDispatcher